sklearn/linear_model/_stochastic_gradient.py

```diff
@@ -1032,9 +1058,17 @@ def predict_log_proba(self):
 
         See ``predict_proba`` for details.
 
+        Side note: ``predict_log_proba`` is a property. Technically, it
+        accepts no arguments except self, and returns a method,
+        ``_predict_log_proba``. However, since it returns a callable
+        method, when it's invoked in code, it appears
+        like it's being called with an argument (of input data) X, and
+        returning probabilities T.
```

Comment:
That is true, but it's an implementation detail.  I thin what is said at the top,

> This method is only available for log loss and modified Huber loss.

Is enough from the user perspective, so I would remove this note.